CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ if (CryptoPP_FOUND)  # vcpkg-provided cmake package called CryptoPP
 else()
     find_package(Crypto++ REQUIRED)
 endif ()
+find_package(fmt REQUIRED)
```

Comment:
This should probably look for 7.1.x or higher.

Reply:
Ubuntu only ships 6.1, and 18.04 ships 4.0

I hate this trash...

Reply:
AFAIK, 7.x has `std::fmt` compatibility, previous versions had issues (because the proposal was changed in standardization).

Reply:
I couldn't find documentation on that, but I locked to the version that the build job runs (6.1.2). I have 7.1.3 here